### DISCUSSION TITLE
Reducing Quest XP for a single student

### DISCUSSION THREAD

**Discussion by cynthia-mantel** [bytedeck/bytedeck#1577] [Q&A] [answered]
I would like to approve a quest for a student but reduce the XP for that one specific submission. For example: when students are partially completing a quest due to absence. (I do not use the XP for grades, but students are trying to accumulate more XP/badges.)
Thanks.

**Comment by tylerecouture** [comment, marked as answer]
Great question!  And thank you for posting it here.

I can suggest a couple mechanisms for customizing the amount of XP a student gets for a specific quest.  Hopefully one of these will work for you:

### 1. Skip the quest and grant a badge instead

Instead of approving the quest, click the "Skip" button, this indicates the student did not complete (or fully complete) the quest and does not grant them XP for the quest, but it does count as complete for prerequisite purposes, so they can move on to the next quest.

Grant them a badge to represent the partial completion.  You may already have a badge that makes sense, of the appropriate amount, or you may need to make one for this purpose. "I'm skipping you through this quest, so you won't get XP for it, but here's a badge for the work you did." Or something like that.

Example:
![image](https://github.com/bytedeck/bytedeck/assets/10604391/bd2cf31e-45fd-4fa1-ab36-3b17506e5c1e)

Skipped quests appear grayed out in their profile and completed quests list:
![image](https://github.com/bytedeck/bytedeck/assets/10604391/fdf3cc07-f891-4e69-9010-3196d630c67d)


### 2. Adjust the students total XP manually

If it is a one off thing you want to easily adjust a student's XP for any reason, you can add an adjustment to their course through their profile like this:

1. From the student's profile, click the edit link beside their course:
![image](https://github.com/bytedeck/bytedeck/assets/10604391/28aa6952-f34e-415a-aedf-cbd6ed245dcf)
2. You can then add n XP adjustment and an optional explanation/reminder:
![image](https://github.com/bytedeck/bytedeck/assets/10604391/4ddc36c3-b949-405a-a93f-3c5ef01d020b)
3. This adjustment and explanation will now appear in their profile under that course:
![image](https://github.com/bytedeck/bytedeck/assets/10604391/f3d18014-d8f1-4230-938a-6ef938046448)
